#168841818 Add UI/Template resposiveness, color constistency, fix logout button

Kagorora opened the pull request:
#### What does this PR do?

- add responsiveness to the UI and maintain color consistency.

#### Description of Task to be completed?

- Enables users to view our system on different screen devices(phone, tablet or laptops)

#### How should this be manually tested?

- via the following link [https://kagorora.github.io/TeamWork/UI/index.html](https://kagorora.github.io/TeamWork/UI/index.html)

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [#168841818](https://www.pivotaltracker.com/n/projects/2398054)

#### Screenshots (if appropriate)

![NEW](https://user-images.githubusercontent.com/48021034/65943923-44336b00-e439-11e9-9545-f5c6f12c3a89.JPG)

![small](https://user-images.githubusercontent.com/48021034/65966418-bbccbe80-e468-11e9-8f1e-341f91b5e670.JPG)

![resp](https://user-images.githubusercontent.com/48021034/65966458-cc7d3480-e468-11e9-8460-8536c28f875c.JPG)


#### Questions: (modifié) 

- N/A

Kagorora committed:
feature(signup): add signup

	- create signup endpoint
	- test signup endpoint

[Finishes: #168682249]

Kagorora committed:
feature(login): Add login

  - create login endpoint
  -  test login endoint

 [Finishes: #168682697]

Kagorora committed:
feature(create Article): Add Create Article

	- create endpoint to create article
	- create  test for create article

[Finishes #168682718]

Kagorora committed:
feature(edit article): add create article

 - add create article endpoint
 - test edit article endpoint

[Finishes #168683783]

Kagorora committed:
feature(delete): add delete article

	- add feature to delete article
	- and it test

[Finishes #168683826]

Kagorora committed:
feature(comment): add comment

	- add comment end point
	- test comment endpoint

[Finishes #168683864]

Kagorora committed:
feature(all articles): dispal all the articles

	- add end point to view all the articles sorted by recently posted
	- test endpoint

[Finishes #168683903]

Kagorora committed:
feature(search): add search article

	- create search article endpoint
	- test search all article endpoint

[Finishes #168683950]

Kagorora committed:
feature(search): Add search by category

	- add search by category
	- test search by category end point

[Finishes #168684714]

Kagorora committed:
feature(flag): add flag article and comment

	- flag articles as inappropriate
	- flag comments as inappropriate
	- test both endpoints

[Finishes #168684716]

Kagorora committed:
bug(code Climate): fix code climate

 - add ignore UI

Kagorora committed:
ft(FlagedArticle): delete Flaged Article

	- create route to delete flaged articles as inappropriate
	- create test for delete articles flaged as inappropriate

[Finishes #168684725]

Kagorora committed:
bg(fix climate): fix code climate

 - fix code climate

[Finishes #168769983]

Kagorora committed:
bug(fix maintenabilty)

 - ignore ui

Kagorora committed:
feature(fix code climate)

	- ignore frontend

Kagorora committed:
chore(Fix test): fix test and status code

	- fix incorect status
	- fix test

[Finishes: #168837916]

Kagorora committed:
chore(fix UI): Add responivess:

 - add responiveness to the UI
 - add color consistency
 - fix non working buttons(logout)

[Finishes: #168841818]

NkFab commented on UI/css/index.css:
```diff
@@ -68,6 +68,8 @@
     width: 100%;
     position: relative;
+    display: flex;
+    /* justify-content: center; */
```
Remove the commented line.

NkFab commented on UI/js/employee.js:
```diff
@@ -29,7 +28,19 @@ const editIcon = document.querySelector('.editIcon');
+}
 
 deleteIcon.addEventListener('click', deleteArticle);
 // function DisplayWarning() {
```
Remove the commented line.

NkFab reviewed (commented):
Remove every commented line that isn't descriptive.

Kagorora committed:
feature(fix UI): add Respomsiveness

 - fix responsivess for devices of screen less than 360px and 414px

 [Finishes: #168841818]

Kagorora committed:
chore(comments): remove comments

 - remove unnecessary

[Finishes: #168841818]
